Constraint/Constraints/GreaterThan.php

```diff
@@ -45,7 +45,7 @@ protected function configureOptions(OptionsResolver $resolver): void
     {
         $resolver
             ->setRequired(['value'])
-            ->setAllowedTypes('value', ['int'])
+            ->setAllowedTypes('value', ['numeric', 'string'])
```

Comment:
You shouldn't allow `string` type here: `numeric` type already handles values like `"5"` or `"5.2"`.

By allowing `string` type, values like `"foo"` would be accepted, which is not supported by Parsley.